charts/radar-self-enrolment-ui/templates/deployment.yaml

```diff
@@ -38,20 +38,34 @@ spec:
           image: {{ template "radar-self-enrolment-ui.image" . }}
           imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
           env:
-          - name: ORY_SDK_URL
+          - name: KRATOS_INTERNAL_URL
```

Comment:
what's implied by internal here? As we are using the public one here

Reply:
Ah yes I think this is the internal URL for the kratos public API and we only need the internal URL since this is just used in the backend.